common/crypto.hpp

```diff
@@ -46,6 +46,10 @@ class PrivateKey {
 public:
 	PrivateKey() {};
 
+	static ExpectedPrivateKey Load(const string &private_key_path) {
+		return Load(private_key_path, "");
+	}
```

Comment:
In the `.cpp` file, we have a similar wrapper:
```
ExpectedPrivateKey PrivateKey::LoadFromPEM(const string &private_key_path) {
	return PrivateKey::LoadFromPEM(private_key_path, "");
}
```

Consider moving the definition of `Load` to the cpp or to bring `LoadFromPEM` to the header :pray: